README.adoc

```diff
@@ -50,7 +50,7 @@ make dockerize
 
 [source, bash]
 ----
-kubectl apply -f config/kafka
+kubectl apply -n riff-system -f config/kafka
```

Comment:
this depends on the namespace being there - we should add instructions to create it like we do https://github.com/projectriff/riff/blob/master/Getting-Started.adoc#install-kafka-chart[here]

Reply:
Can you add that one line, or do you want me to do it and merge?

Reply:
Sure, I can add it on merge

Reply:
Also adding `-n riff-system` to the teardown script